packages/worker/src/tests/TestConfiguration.ts

```diff
@@ -36,6 +36,7 @@ import {
 } from "@budibase/types"
 import API from "./api"
 import jwt, { Secret } from "jsonwebtoken"
+import { cloneDeep } from "lodash"
```

Comment:
Can we use `import cloneDeep from "lodash/fp/cloneDeep"` for a better tree shaking and hoisting?

Reply:
I understand what tree shaking is and how this would help, how does this help with hoisting? I only have a rough idea of what hoisting is 🙈